File: Programming-languages-and-laboratory/Python/Exercises/05_compito_java_20150715/Toner.py
from Prodotto import Prodotto


class Toner(Prodotto):
    def __init__(self, tipo, codice, modello, produttore, giorno, mese, anno, prezzo, modello_stampante):
        super().__init__(tipo, codice, modello, produttore, giorno, mese, anno, prezzo)
        self._modello_stampante = modello_stampante

    def __str__(self):
        return "Toner\t" + super().__str__() + "\t" + self._modello_stampante + "\t-"

File: II - Linguaggi di programmazione e laboratorio/Esame/Java/05_compito_java_2015_07_15/Toner.java
import java.util.*;

public class Toner extends Prodotto {
    private String modelloStampante;

    public Toner(int codice, String modello, String produttore, int giorno, int mese, int anno, int prezzo, String modelloStampante) {
        super(codice, modello, produttore, giorno, mese, anno, prezzo);
        this.modelloStampante = modelloStampante;
    }

    public String toString(){
        return "Toner\t" + codice + "\t" + modello + "\t" + produttore + "\t" + giorno + "\t" + mese + "\t" + anno + "\t" + prezzo + "\t" + modelloStampante + "\t-";
    }
}